package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mapbox/vtcomposite",
-  "version": "0.5.1",
+  "version": "0.6.0-dev1",
```

Comment:
I assume the version update will no longer have `dev1`  once the changes are reviewed/tested?

Reply:
yep definitely - I used dev1 just to test downstream, and released it under the `dev` tag in NPM.